apps/blink/main.c

```diff
@@ -1,12 +1,57 @@
 #include <firestorm.h>
```

Comment:
We don't want this merged into master. There is now a `blink_sync` app in master that does this (only blinks one LED).

The `blink` app uses a timer, and `blink_sync` uses a delay function like in this change.